openapi.yml

```diff
@@ -146,19 +163,69 @@ components:
       format: FiscalCode
       x-import: italia-ts-commons/lib/strings
       example: SPNDNL80R13C555X
+    ArrayIndexUpTo4:
+      type: integer
+      minimum: -1
+      maximum: 5
+      exclusiveMaximum: true
+    ExistingArrayIndexUpTo4:
+      type: integer
+      minimum: 0
+      maximum: 5
+      exclusiveMaximum: true
+    NonEmptyString:
+      type: string
+      format: NonEmptyString
+      x-import: italia-ts-commons/lib/strings
     OrganizationFiscalCode:
       type: string
       description: Organization's fiscal code.
       format: OrganizationFiscalCode
       x-import: italia-ts-commons/lib/strings
       example: 01234567890
+    OrganizationScope:
+      type: string
```

Comment:
Ask @cloudify if we use `enum` or `x-extensible-enum` like it's done in https://github.com/teamdigitale/io-functions-commons/blob/master/openapi/definitions.yaml#L37